src/main/java/net/pms/dlna/LibMediaInfoParser.java

```diff
@@ -678,13 +688,39 @@ public static String getAvcLevel(String value) {
 		// High@L3.0
 		// High@L4.0
 		// High@L4.1
-		final String avcLevel = substringAfterLast(lowerCase(value), "@l");
+		// // Value can look like "Stereo High@L4.1 / High@L4.1".
+		if (isNotBlank(substringBetween(lowerCase(value), "@l" , " /"))) {
+			final String avcLevel = substringBetween(lowerCase(value), "@l" , " /");
+		} else {
```

Comment:
@Sami32 ```final``` means it can only be set once. However, the bigger problem here is scope. ```avcLevel``` will only exist in the scope of that if-block, so it won't exist as soon as the code leaves that.